### ISSUE TITLE
Issue with the "date" type in an @Example decorator

### ISSUE THREAD

**Issue by electrotype** [lukeautry/tsoa#36]
I tried to annotate the method of a controller using :

```
@Example<UpperResult>({
    name: "test",
    upperName: "TEST",
    date: new Date(),
})
```

But the `date` part doesn't seem to be generated in the "swagger.json" file. 

Any idea?

**Comment by lukeautry** [comment]
Looks like we'd need to add something here to handle that: https://github.com/lukeautry/tsoa/blob/master/src/metadataGeneration/methodGenerator.ts#L117

Shouldn't be too difficult.

**Comment by egandro** [comment]
Thx for pointing me here :)

I'll have a look at this.

**Comment by lukeautry** [comment]
@electrotype I've thought more about this and I'm wondering if it's a bad idea to introduce the concept of "expressions" in the example

**Comment by isman-usoh** [comment]
Fix: 61573badd73ce30f0ce9714acfdd948802f5bf7b
